Robot Code/src/org/usfirst/frc/team3316/robot/subsystems/Hood.java

```diff
@@ -0,0 +1,49 @@
+package org.usfirst.frc.team3316.robot.subsystems;
+
+import org.usfirst.frc.team3316.robot.Robot;
+import org.usfirst.frc.team3316.robot.robotIO.DBugSpeedController;
+
+import edu.wpi.first.wpilibj.AnalogPotentiometer;
+
+public class Hood extends DBugSubsystemCC
+{
+	private DBugSpeedController hoodMotor;
+	private AnalogPotentiometer hoodPot;
+
+	public Hood()
+	{
+		hoodMotor = Robot.actuators.hoodMotor;
+		addSpeedController(hoodMotor);
+
+		hoodPot = Robot.sensors.hoodPot;
+	}
+
+	public void initDefaultCommand()
+	{
+	}
+
+	/**
+	 * Set the voltage for the motors of this subsystem.
+	 * @par v - The voltage for the motors (between 1.0 to -1.0).
+	 */
+	public boolean setMotors(double v)
+	{
+		// If the hood is able to move.
+		if (getAngle() > (double) config.get("hood_Pot_LowThresh")
```

Comment:
What if a command sets voltage while we're in a valid position, but then nothing interrupts this until we're running past the final position. Can we have some sort of watchdog which sets the motors to 0 if they reach past the limits and are moving in the wrong direction?